src/com/itacademy/eshop/product/Description.java

```diff
@@ -0,0 +1,31 @@
+package com.itacademy.eshop.product;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Description {
+
+    private String productDescription;
+    private String descriptionAdditionTime;
+
+    public Description() {}
```

Comment:
If you have no other constructors defined the no args constructor if specified by default and you do not need to write it yourself.